storybookjs/storybook
# Using svg-url-loader for webpack configs that accept svgs

Issue: #2474 

## What I did
Added `svg-url-loader` to make svg imports smaller

## How to test
build core, docs, examples

Is this testable with jest or storyshots?
No
Does this need a new example in the kitchen sink apps?
No
Does this need an update to the documentation?
No
If your answer is yes to any of these, please make sure to include it in your PR.
